.idea/misc.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="..\:/Users/judem/Git/Euphony-io/sound-helper-eu/app/src/main/res/layout/activity_main.xml" value="0.25364583333333335" />
+        <entry key="..\:/Users/judem/Git/Euphony-io/sound-helper-eu/app/src/main/res/layout/activity_transmit.xml" value="0.25364583333333335" />
```

Comment:
This path appears to be user-dependent. How about adding .idea/misc.xml to .gitignore?

Reply:
Added misc.xml to .gitignore!